.ci/Jenkinsfile

```diff
@@ -59,6 +60,7 @@ pipeline {
         cleanup()
         withGoEnv(){
           dir("${BASE_DIR}"){
+            retryWithSleep(retries: 2, seconds: 5, backoff: true){ sh(label: "Install Docker", script: '.ci/scripts/install-docker-compose.sh') }
```

Comment:
Temporary workaround as long as the CI Workers don't have docker-composer with version > 1.25